com/avgustin/lesson1/Homework1.java

```diff
@@ -0,0 +1,7 @@
+package com.avgustin.lesson1;
+
+public class Homework1 {
+    public void myMethod1() {
```

Comment:
Можно метод сделать static, чтобы не создавать для него объект
```suggestion
    public **static** void myMethod1() {
```